### DISCUSSION TITLE
Annotating Multimodal Data

### DISCUSSION THREAD

**Discussion by Abigail575** [stuart-lab/signac#1213] [Q&A] [answered]
Hi! 
I'm following the multiomics tutorial and was wondering if you have a multimodal reference dataset for the pancreas that can be used just as you do for the mapping and annotating query datasets tutorial ("panc8") on SeuratDisk? If not, is there a way I can annotate the data from known cell markers from the clustering tutorial that would work for multimodal data? 
Thanks!

**Comment by timoast** [comment, marked as answer]
Hi, there are several annotated reference datasets available here, including Human pancreas: https://azimuth.hubmapconsortium.org/

**Reply by Abigail575** [reply]
Thank you!

**Comment by JoGraesslin** [comment]
What would be a way to annotate cell cluster without a reference dataset?

**Reply by timoast** [reply]
This would usually involve finding differentially accessible peaks or expressed genes between clusters, interpreting what cell types those peaks or genes are active/expressed in, and manually annotating cell types. See the [Signac PBMC vignette](https://stuartlab.org/signac/articles/pbmc_vignette.html#find-differentially-accessible-peaks-between-clusters) and Seurat [PBMC vignette](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html#assigning-cell-type-identity-to-clusters-1) for more information
